src/Sulu/Bundle/AdminBundle/Resources/js/stores/MetadataStore/MetadataStore.js

```diff
@@ -1,24 +1,62 @@
 // @flow
-import {Requester} from '../../services';
+import symfonyRouting from 'fos-jsrouting/router';
+import {buildQueryString} from '../../utils/Request';
+
+const defaultOptions = {
+    credentials: 'same-origin',
+    headers: {
+        'Content-Type': 'application/json',
+        'X-Requested-With': 'XMLHttpRequest',
+    },
+};
 
 class MetadataStore {
-    endpoint: string;
-    metadataPromises: {[string]: {[string]: Promise<Object>}} = {};
+    metadataPromises: {[string]: {[string]: ?Promise<Object>}} = {};
+
+    loadMetadata(type: string, key: string, metadataOptions: Object = {}): Promise<Object> {
+        const parameters = {
+            type: type,
+            key: key,
+            ...metadataOptions,
+        };
 
-    loadMetadata(type: string, key: string): Promise<Object> {
         if (!this.metadataPromises[type]) {
             this.metadataPromises[type] = {};
         }
+        const keyWithOptions = buildQueryString({
```

Comment:
I think that should be something like `key + buildQueryString(metadataOptions)`. Otherwise we are probably running into problems if `metadataOptions` contains an option with the key `key`.